pkg/kubeadm/cmd/init.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/renstrom/dedent"
+	"github.com/spf13/cobra"
+
+	kubeadmapi "k8s.io/kubernetes/pkg/kubeadm/api"
+	kubemaster "k8s.io/kubernetes/pkg/kubeadm/master"
+	kubeadmutil "k8s.io/kubernetes/pkg/kubeadm/util"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+var (
+	init_done_msgf = dedent.Dedent(`
```

Comment:
Consider: report any place where we are writing out files in to the system.  Things like certificates and manifests and kubeconfig.  This'll give some indication of what to clean up if you don't like what we've done.